app/views/spree/admin/permission_sets/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :page_title do %>
+  <%= Spree.t(:permission_sets) %>
+<% end %>
+
+<% if can? :create, Spree::PermissionSet %>
+  <% content_for :page_actions do %>
+    <%= button_link_to Spree.t(:new_permission_set), new_object_url, class: 'btn btn-success', icon: 'add', id: 'admi_new_permission_set_link' %>
+  <% end %>
+<% end %>
+
+<% if @permission_sets.present? %>
+  <table class="table">
+    <thead>
+      <tr data-hook="admin_permission_sets_index_headers">
+        <th><%= Spree.t(:permission_set_name)%></th>
```

Comment:
@nimish-mehta Lets also display update_at timestamp on the permission sets listing page.